cypress/utils/utils.ts

```diff
@@ -668,7 +668,43 @@ export function verifyImportErrorMsg(errorMsg: any): void {
     }
 }
 
+export function migration_wave_kebab_menu(menu): void {
```

Comment:
This method can be used in all kebab menu like in application and analysis page as well .We can rename this to 

export function click_main_kebab_menu_item(menu_item):

Eventually we can replace application menu method with this one .

Reply:
We already have export function application_inventory_kebab_menu(menu, tab?)  for that purpose.